examples/space.html

```diff
@@ -25,7 +26,9 @@
 <body>
   <vr-scene>
     <vr-camera fov="45" near="1" far="2000" y="50"></vr-camera>
-    <vr-cube z="-500" width="50" height="50" depth="50" y="25"></vr-cube>
+    <vr-cube z="-500" width="50" height="50" depth="50" y="25">
+      <vr-audio src="cricket.mp3" autoplay loop></vr-audio>
+    </vr-cube>
     <vr-cube z="-300" width="50" height="50" depth="50" y="25" x="100"></vr-cube>
```

Comment:
What about removing the rest of the cubes so it's clear where the sound is supposed to come from? I haven't tested the scene to see if it feels right. What do you think?